### ISSUE TITLE
PROP #3: Allocate $8000 USD worth LPFi for LPFi-PAI bonding on Penguin Finance

### ISSUE THREAD

**Issue by aperire** [LP-Finance-Inc/governance#3]
# PROP #3: Allocate $8000 USD worth LPFi for LPFi-PAI bonding on Penguin Finance

Date: Jan.26th 2023
Author: Joohwan Lee
Status: Proposed

## Summary
I propose to allocate $8000 USD worth of LPFi to create a bonding pool for LPFi-PAI on Penguin Finance. 
$8000 USD LPFi ≈ 213350 LPFi (≈4.8% of total supply)

## Motivation
Penguin Finance recently partnered with Parrot Finance to provide grants for Bonding, which in summary works like this.
1. LP Finance DAO allocated $8000 USD worth LPFi as bonding rewards.
2. Parrot Finance purchases 2 weeks worth of bonding ($2000) if we set up 8 weeks bond ($1000/week)

![1_WMIqu2z2S3EVe8hxWTVtJw](https://user-images.githubusercontent.com/27561419/233942452-8226451c-0015-4090-82d5-e25f5bfd8008.png)

This would allow LPFi token's liquidity to significantly improve allowing better experience for users.

For details on grants,
https://pngfi.medium.com/penguin-finance-marketplace-on-neptune-61e8dfac387e

## Actions
1. Allocate $8000 USD worth LPFi to Penguin Finance bonding pool

**Comment by TomintheToon** [comment]
Do the trading fee earnings accumulate to the LPfi DAO? Looks like a good idea to me.

**Comment by aperire** [comment]
Yes! And also LPFi liquidity would significantly in support from Parrot Finance.
